Frontend/ui-customer/src/Services/Backend.purs

```diff
@@ -453,6 +453,29 @@ makeUpdateProfileRequest name gender referralCode =
             "BN_IN" -> "BENGALI"
             "TA_IN" -> "TAMIL"
             _       -> "ENGLISH"
+        , clientVersion : Nothing
+        , bundleVersion : Nothing
+    }
+
+makeUpdateVersionRequest :: Version -> Version -> UpdateProfileReq
+makeUpdateVersionRequest clientVersion bundleVersion = 
```

Comment:
if some fields are not required and their type is `Maybe` you can remove such field instead of passing `Nothing`